build/konflux-patch.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+set -e
+
+# Gatekeeper Operator image
+operator_img="registry.redhat.io/gatekeeper/gatekeeper-rhel9-operator@sha256:6e386be134d928bdb03b702e399c97e7aedecacaa3d0813183a8c5ecf13c7bc2"
+# Gatekeeper image
+gatekeeper_img="registry.redhat.io/gatekeeper/gatekeeper-rhel9@sha256:3095f68c12c5dc3b00ce84e1c37d516d96cbcb06d42eaef5372358786956bd62"
```

Comment:
Will these get updated by nudges?

Reply:
That's correct--that's the next piece I need to work out. That and the FBC.